XamarinCommunityToolkit/Views/SnackBar/Options/ActionOptions.shared.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views.Options
+{
+	/// <summary>
+	/// Use for any actions: toasts, SnackBars etc.
+	/// </summary>
+	public class ActionOptions
```

Comment:
Can we make this internal? It seems only the `SnackBarActionOptions` inheritance is used right now, right? If we find a use to actually allow users to use this object we can open it, but for now I would say keep it internal until we actually have use for it to be public.

Reply:
I added extension method ToastAsync
I will add it as a parameter.
Give me a minute

Reply:
done